src/style.css

```
@@ -0,0 +1,40 @@
+body,
+div {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  block-size: cover;
+  background-position: center;
+  background-image: url("images/star.svg");
+}
+
+.stars.stars--1 :nth-child(-n + 1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--2 :nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--3 :nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--4 :nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars.stars--5 :nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}
```

combine these selectors
```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
background-image: url("images/star-active.svg");
}
```